packages/jest-snapshot/src/index.js

```diff
@@ -48,6 +48,10 @@ module.exports = {
     state.getCounter = null;
     state.incrementCounter = null;
     state.snapshot = SnapshotFile.forFile(filePath);
+    state.snapshotsAdded = 0;
+    state.snapshotsRemoved = 0;
+    state.snapshotsMatched = 0;
```

Comment:
just for you to think about: is there really a point in mentioning `snapshots` in the `snapshot state`? You end up repeating the word snapshot twice and the first one already signals to the reader of the code that it is about snapshot:

`snapshotState.snapshotsAdded` vs. `snapshotState.added` – the first one seems to have some redundancies.